frontend/national-geo-spatial-data-explorer/style.css

```
@@ -0,0 +1,631 @@
+    --border-dark: #334155;
+    --accent-emerald: #10b981;
+    --accent-amber: #f59e0b;
+    --shadow-sm: 0 1px 3px rgba(0,0,0,0.1);
+    --shadow-md: 0 4px 6px -1px rgba(0,0,0,0.1);
+    --shadow-lg: 0 10px 15px -3px rgba(0,0,0,0.1);
+}
+
+body {
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)